home.css

```diff
@@ -0,0 +1,122 @@
+body{
+    background-color: var(--st-white);
+}
+.flex-spacearound{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    gap:1.5rem;
+}
+.bold-subheading{
+    font-weight: 1000;
+    padding:1.5rem 0rem;
+}
+input.search-input {
+    min-width: 19rem;
+    border-radius: 4px;
+    display: block;
+    padding: 9px 4px 9px 40px;
```

Comment:
You can prefer to use rem instead of pixels here. Other than that code looks clean.